command/CommandController.js

```diff
@@ -119,7 +127,8 @@ router.post('/article', VerifyToken, async function(req, res) {
     const result = await processArticleRequest(
       req,
       false,
-      req.body.extendedData == true
+      req.body.extendedData == true,
```

Comment:
What is this extendedData flag for?  I know it came in the last patch, but trying to figure out what it means from the name.

Reply:
It is to return the Favicon and the Publisher name (as we make more calls to fetch them). They are used by the iOS app